### PR TITLE
add local development setup docs

### PR DESCRIPTION
**Pull request by philbrookes**
add local development setup docs.

I have been struggling to verify this locally using:

```
 ./bin/build.sh local-site.yml
```
or
```
 ./bin/quick-build.sh local-site.yml
```

These scripts are complaining about formatting in unrelated files and will not serve a local copy of the site - even if I fix the issues it does not like - if there are suggestions to get this working, please let me know.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -55,6 +55,8 @@ include::{partialsdir}/attributes.adoc[]
 ** xref:testcard.adoc[{custom-service} for the Showcase App]
 ** xref:testcard.adoc[{unifiedpush-service} for the Showcase App]
 
+* xref:local_development.adoc[Setting up a Local Development Environment]
```
Comment (pb82) [inline modules/ROOT/nav.adoc]: Should this be a new section or become `Getting started > Installation`?

File: modules/ROOT/pages/local_development.adoc [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+The OpenShift client tools are a requirement and must be 3.9 or greater,
+they can be downloaded from 
+link:https://www.openshift.org/download.html[here].
+
+=== Mobile-core Installer
+
+We configure the local development installation of {product-name}
+using ansible scripts in our link:https://github.com/aerogear/mobile-core[mobile-core] repo.
+
+Clone this repo to your local machine and check out the 1.0.0-alpha tag using:
```
Comment (pb82) [inline modules/ROOT/pages/local_development.adoc]: will it still be `1.0.0-alpha` after the community release or will everything be retagged with `1.0.0`?

Comment (wtrocki) [reply modules/ROOT/pages/local_development.adoc]: Best to use variable here :D

File: modules/ROOT/pages/local_development.adoc [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+are installed and the right version.
+
+Once verified it prompts you for your docker hub login credentials, which
+must be valid for the installation to continue.
+
+The next set of values are optional and can be left with the default values:
+```
+DockerHub Tag (Defaults to 1.0.0-alpha): 
+DockerHub Organisation (Defaults to aerogearcatalog): 
+Cluster IP (Defaults to 192.168.37.1):
```
Comment (wtrocki) [inline modules/ROOT/pages/local_development.adoc]: Optional note: point cluster to external network ip so mobile devices could see it.

File: modules/ROOT/pages/local_development.adoc [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+=== Docker
+
+Docker is a requirement, and must be a specific version depending
+on your OS in order to be compatible with the latest release of
+OpenShift.
+
+==== Darwin / MacOSX
+
+On Darwin / MacOSX operating systems, you may install the latest version of
```
Comment (wtrocki) [inline modules/ROOT/pages/local_development.adoc]: I could not get that running with latest. Same as for linux we may use older version. https://github.com/darahayes/oc-mobile-up/#troubleshooting

**Comment by pb82** [comment]
LGTM

File: modules/ROOT/pages/local_development.adoc [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+=== Ansible
+
+Ansible version 2.3 or later is a requirement of running the 
+installation scripts on your local machine, follow the 
+installation steps
+link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[here].
+
+=== Node.js
+
+Node.js version 8 or later and the bundled package manager 'npm'
```
Comment (maleck13) [inline modules/ROOT/pages/local_development.adoc]: is this actually still needed? I know we had it for when the UI was an extension, but what is it used for now?

File: modules/ROOT/pages/local_development.adoc [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+Clone this repo to your local machine and check out the {release-number} tag using:
+```
+git clone git@github.com:aerogear/mobile-core.git
+git checkout {release-number}
+```
+
+=== Docker
+
+Docker is a requirement, and must be version 17.09.1 in order to be compatible with
```
Comment (maleck13) [inline modules/ROOT/pages/local_development.adoc]: I think this is only for linux? My mac has 18.03 and is fine

File: modules/ROOT/pages/local_development.adoc [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+=== Docker
+
+Docker is a requirement, and must be version 17.09.1 in order to be compatible with 
+the latest release of OpenShift, depending on your OS this is downloaded from different places:
+
+==== Darwin / MacOSX
+
+On Darwin / MacOSX operating systems, you must install version
```
Comment (maleck13) [inline modules/ROOT/pages/local_development.adoc]: same for this had no issues with 18

File: modules/ROOT/pages/_partials/attributes.adoc [hunk 8]
```diff
@@ -2,7 +2,9 @@
 
 :product-name: Mobile Services
 
-:service-name:
+:release-number: 1.0.0-alpha
```
[the remark this answers is not in the document]
Comment (finp) [reply modules/ROOT/pages/_partials/attributes.adoc]: ah, it's the installer release no.

**Pull request merged**